### PR TITLE
Add a release checklist

### PR DESCRIPTION
**Pull request by seisman**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: Release checklist
+about: Checklist for a new release
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+- [ ] Document all important changes in Changelog.md
```
Comment (core-man) [inline .github/ISSUE_TEMPLATE/release-checklist.md:10]:
```suggestion
- [ ] Document all the important changes in Changelog.md
```

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+- [ ] Set the release date in Changelog.md
+- [ ] Set the current release version and data in Changelog.md
```
Comment (core-man) [inline .github/ISSUE_TEMPLATE/release-checklist.md:12]: We don't have the current release version and data in Changelog.md

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+about: Checklist for a new release
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+- [ ] Document all the important changes in Changelog.md
+- [ ] Set the release date in Changelog.md
+- [ ] Set the current release version and data in Changelog.md
```
Comment (seisman) [inline .github/ISSUE_TEMPLATE/release-checklist.md:12]:
```suggestion
- [ ] Set the current release version and data in README.md
```

File: .github/ISSUE_TEMPLATE/release-checklist.md (hunk 2 above)
Comment (seisman) [reply .github/ISSUE_TEMPLATE/release-checklist.md:12]: Typo. Should be "README.md"

File: .github/ISSUE_TEMPLATE/release-checklist.md (hunk 3 above)
Comment (core-man) [reply .github/ISSUE_TEMPLATE/release-checklist.md:12]: How can we set current release data?

Comment (seisman) [reply .github/ISSUE_TEMPLATE/release-checklist.md:12]: When we decide to release v0.2.0, we need to change the current version to v0.2.0 (2021-06-08) in README.md before making the release.

Comment (core-man) [reply .github/ISSUE_TEMPLATE/release-checklist.md:12]: So it's `Set the current release version in README.md`?

Comment (seisman) [reply .github/ISSUE_TEMPLATE/release-checklist.md:12]: Typo again. I meant "date".

Comment (seisman) [reply .github/ISSUE_TEMPLATE/release-checklist.md:12]: image

**Review by core-man — APPROVED** [review]

**Pull request merged**